pkg/crc/machine/machine.go

```diff
@@ -185,8 +185,11 @@ func Start(startConfig StartConfig) (StartResult, error) {
 			return *result, errors.Newf("Error loading bundle metadata: %v", err)
 		}
 		if bundleName != filepath.Base(startConfig.BundlePath) {
-			logging.Fatalf("Bundle '%s' was requested, but the existing VM is using '%s'",
+			logging.Debugf("Bundle '%s' was requested, but the existing VM is using '%s'",
 				filepath.Base(startConfig.BundlePath), bundleName)
+			result.Error = fmt.Sprintf("Bundle '%s' was requested, but the existing VM is using '%s'",
+				filepath.Base(startConfig.BundlePath), bundleName)
+			return *result, errors.Newf("Found a CodeReady Containers VM using bundle %s", filepath.Base(startConfig.BundlePath))
```

Comment:
Is it intentional that the returned error is not "Bundle %s was requested but ..." ?

Reply:
Yes, i thought normal error would show a brief info and the debug log would give the details with the bundle names.

Reply:
If you think this needs to be changed to "Bundle %s was requested but ..." that is fine with me, it does make the message more clear.

Reply:
I'm worried about what the user-visible message will be. At the moment it's "Bundle '%s' was requested, but the existing VM is using '%s'", which is somehow explicit as to what the problem is. If with this change we show "Found a CodeReady Containers VM using bundle %s", imo this is less explicit.

Reply:
Okay, let me change it back to the original message..